src/ltc.c

```diff
@@ -251,14 +251,18 @@ void ltc_encoder_set_filter(LTCEncoder *e, double rise_time) {
 }
 
 int ltc_encoder_set_bufsize(LTCEncoder *e, double sample_rate, double fps) {
-	free (e->buf);
-	e->offset = 0;
-	e->bufsize = 1 + ceil(sample_rate / fps);
-	e->buf = (ltcsnd_sample_t*) calloc(e->bufsize, sizeof(ltcsnd_sample_t));
-	if (!e->buf) {
-		return -1;
-	}
-	return 0;
+	return ltc_encoder_set_buffersize(e, sample_rate, fps);
+}
+
+int ltc_encoder_set_buffersize(LTCEncoder *e, double sample_rate, double fps) {
+    free (e->buf);
```

Comment:
Please use tabs for indent, space to align. Prefer to keep whitespace changes separately. It makes it hard to see the actual diff here.

I should probably push the whole thing though clang-format at some point.

Reply:
Sorry about that. I have my compiler (Xcode) setup to turn tabs into spaces automatically. Many people don't want tabs in their code. Be aware that filter and volume getters have also been affected by different whitespace formatting. I only realized it when I looked at the code online. Again sorry about that.

Yes, doing a clang-format is a good idea indeed.

Reply:
The convention you are following is to indent with one tab = 8 spaces, correct?